docs/reference/known_limitations.md

```diff
@@ -15,25 +15,35 @@ We are developing a new feature that will allow you to modify these settings dir
 
 ## Backups
 
-The limitations related to backups in Percona Everest are as follows:
+Let's delve into the limitations of on-demand backups in Percona Everest. 
+
+### PostgreSQL limitations for on-demand backups
 
 - When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 
 
     **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.
 
-- You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
+- You cannot change the bucket name and region for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
+
+- If you create more than three backup storages, you can use any of the existing storages across on-demand backups and schedules, as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.
 
+    If you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.
 
 
 ## Scheduled backups
 
-Due to an issue that has been identified in PostgreSQL editing/deletion of schedules, the following functionality is temporarily unavailable for PostgreSQL:
+Let's explore the constraints of scheduled backups in Percona Everest.
 
-- Modifying the storage location in existing schedules
-- Removing existing schedules
-- Utilizing the same storage location for different schedules
+### PostgreSQL Limitations for schedules
 
-The bug may result in a situation where editing or deleting the schedules could corrupt previously taken backups, making it impossible to restore from them.
+Due to PostgreSQL limitations, the following functionality is unavailable for PostgreSQL:
+
+- Modifying the storage location in existing schedules
+- Using the same backup storage for different schedules
+- Creating more than three schedules for PostgreSQL
+- Using more than three different backup storages in total, including those used in existing backup schedules.
```

Comment:
Since we are in the schedules section wouldn't it make more sense to refer to the on-demand ones instead?
```suggestion
- Using more than three different backup storages in total, including those used in existing on-demand backups.
```